scripts/publish_sandbox.sh

```diff
@@ -11,4 +11,4 @@ fi
 
 ./scripts/build_layers.sh
 ./scripts/sign_layers.sh sandbox
-VERSION=$VERSION REGIONS=sa-east-1 ./scripts/publish_layers.sh
+VERSION=$VERSION REGIONS=us-west-1 ./scripts/publish_layers.sh
```

Comment:
Maybe you didn't want to commit this file?

Reply:
Ahh whoops, good catch